qm-windowmanager/usr/lib/tmpfiles.d/etc/containers/systemd/weston_terminal.container

```diff
@@ -0,0 +1,19 @@
+[Unit]
+After=mutter.service
+Description=weston_terminal container
```

Comment:
Weston terminal was just an example workload to have something printed in the screen. Otherwise, we just get a plain background. But as a package, I think we probably want an empty screen where users put their own workloads.

Reply:
thanks for the explanation, I will document that as well as we talked. :)